### PR TITLE
Change to spatial data type

### PR DESCRIPTION
**Pull request by s3364859**
@6nop @s3364942 @s3494110 @s3482247 

In this PR:
 - Removed Location.LocalGovernmentAreaBoundary
 - Updated the KML parser
 - Some tidying and housekeeping

1294314068_cleaning-dog

### CODE REVIEW

File: CPT331.Data.Migration/Migrations/Location.LocalGovernmentArea.sql [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+SET ANSI_PADDING ON
+GO
+
+CREATE TABLE [Location].[LocalGovernmentArea]
+(
+	[ID] [int] IDENTITY(1,1) NOT NULL,
+	[StateID] [int] NOT NULL,
+	[Name] [varchar](100) NOT NULL,
+	[Area] [geography] NULL,
```
Comment (s3364859) [inline CPT331.Data.Migration/Migrations/Location.LocalGovernmentArea.sql]: New geography column.

File: CPT331.Data.Migration/Migrations/M0001.cs [hunk 2]
```diff
@@ -221,6 +147,9 @@ public override void Up()
 						.NotNullable()
 						.WithDefault(SystemMethods.CurrentUTCDateTime);
 
+			//	Tables
+			Execute.EmbeddedScript("Location.LocalGovernmentArea.sql");
```
Comment (s3364859) [inline CPT331.Data.Migration/Migrations/M0001.cs]: Need this because FluentMigrator is incomplete.

File: CPT331.Data.Migration/Program.cs [hunk 3]
```diff
@@ -37,6 +37,20 @@ static Program()
...
 		private static readonly Options _options;
 
+		private static void CheckGeography(SqlConnection sqlConnection)
+		{
+			OutputStreams.WriteLine($"Checking geography...");
+
+			StringBuilder stringBuilder = new StringBuilder();
+
+			//	Automatically correct any geography nasties - this could cause some geography to shift slightly
+			stringBuilder.AppendLine("UPDATE Location.LocalGovernmentArea SET Area = Area.MakeValid() WHERE Area.STIsValid() = 0;");
```
Comment (s3364859) [inline CPT331.Data.Migration/Program.cs]: Looks like a hack, but is actually required to fix any geography that might be weird.

File: CPT331.Data.Migration/Migrations/M0001.cs (hunk 2 above)
Comment (6nop) [inline CPT331.Data.Migration/Migrations/M0001.cs]: @s3364859 Carl,

Can we drop the [Location].[Location] table?

**Comment by 6nop** [comment]
I have a feeling the automated build may fail. The unit tests seem to need a DB, which is not available on the current build server.

This will probably be resolved when I setup the build agent on the server.

Otherwise all good 👍

**Pull request merged**